docs/installation/helm.md

```diff
@@ -119,25 +206,34 @@ To run tests, try:
   $ helm test siglens
 ...
 ```
-#### Step 6: Accessing SigLens
+
```

Comment:
Add one more step here to ensure that SigLens is running, by checking the relevant pods and services